info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -833,6 +840,7 @@ Exceptions cases:
 4) Supertype lists: \
    a) 4 spaces if colon before supertype list is on new line \
    b) 4 spaces before each supertype; 8 if colon is on new line
+5) When statements - 8 spaces are used to indent condition branches: (`CONDITION -> foo()`) 
```

Comment:
not supported in diktat? - need to create an issue

Reply:
Do you mean indentation inside block that follows an arrow, or if single statement starts on a new line? Like
```
condition -> {
        // indent with 8 spaces
        foo()
        bar()
}
```
This case should be covered: both opening braces increase indentation by 4 spaces.

Reply:
no, my bad, it is a bug